components/locale/macros/src/lib.rs

```diff
@@ -0,0 +1,192 @@
+mod token_stream;
+
+extern crate proc_macro;
+
+use proc_macro::TokenStream;
+use token_stream::IntoTokenStream;
+
+fn get_value_from_token_stream(input: TokenStream) -> String {
```

Comment:
Looking for literal `"` is also what you do in tinystr.  We need to fix this, but it shouldn't block this PR.  https://github.com/zbraniecki/tinystr/issues/24